__tests__/app.test.js

```diff
@@ -158,6 +158,22 @@ describe("/api/articles", () => {
         });
       });
   });
+  it("GET 200: Accepts a topic query to filter articles by the topic value specified in the query", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(12);
+      });
+  });
+  it("GET 200: Responds with an empty array if passed in a query of non-existent topic", () => {
```

Comment:
I think you should get a 200 if the topic exists but there aren't any articles for it yet. For a topic that doesn't exist you should get a 404